OpenRA.Game/Traits/World/CountryGroup.cs

```diff
@@ -0,0 +1,28 @@
+﻿// #region Copyright & License Information
+// /*
+//  * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+//  * This file is part of OpenRA, which is free software. It is made
+//  * available to you under the terms of the GNU General Public License
+//  * as published by the Free Software Foundation. For more information,
+//  * see COPYING.
+//  */
+// #endregion
```

Comment:
The `// ` are a bit strange in front of multi-line comments. I know that MonoDevelop does them when you use a templated header which seems to be a bug.